test/distributed_ranges/include/common_tests.hpp

```diff
@@ -171,13 +166,13 @@ bool is_equal(std::forward_iterator auto it, rng::range auto &&r) {
 auto equal_message(rng::range auto &&ref, rng::range auto &&actual,
                    std::string title = " ") {
   if (is_equal(ref, actual)) {
-    return drfmt::format("");
+    return "";
   }
   return drfmt::format("\n{}"
-                     "    ref:    {}\n"
-                     "    actual: {}\n  ",
-                     title == "" ? "" : "    " + title + "\n",
-                     rng::views::all(ref), rng::views::all(actual));
+                       "    ref:    {}\n"
```

Comment:
only formatting change? and did it pass checks in changes shape?
did you run format checks?